controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -769,5 +777,37 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
     setGraphRouterCompatibilityVersion: (req, ctx) => {
       return setGraphRouterCompatibilityVersion(opts, req, ctx);
     },
+
+    getProposalsOfFederatedGraph: (req, ctx) => {
+      return getProposalsOfFederatedGraph(opts, req, ctx);
+    },
+
+    getChecksOfProposal: (req, ctx) => {
```

Comment:
`for`, `of` `getX` do we have a pattern here or can we follow one?

Reply:
We use "for" for mutations, we use "By" for things like this, ex: getChecksByFederatedGraph.
Should I go for getChecksByProposal or getProposalChecks?

Reply:
getProposalChecks sounds less verbose